src/rpc/net.cpp

```diff
@@ -45,6 +45,37 @@ UniValue getconnectioncount(const JSONRPCRequest& request)
     return (int)g_connman->GetNodeCount(CConnman::CONNECTIONS_ALL);
 }
 
+UniValue setmaxconnections(const JSONRPCRequest& request)
+{
+    int newMaxCount = 0;
+    const std::string minConnCount = to_string(MAX_ADDNODE_CONNECTIONS);
+
+    if (request.fHelp || request.params.size() != 1)
+        throw runtime_error(
+            "setmaxconnections\n"
+            "\nSets the maximum number of connections to other nodes.\n"
+            "\nArguments:\n"
+            "1. \"maxconnectioncount\"     (numeric, required) The new maximum connection count (must be >= " + minConnCount + ")\n"
```

Comment:
I think we don't need the quotes around maxconnectioncount, since it's a number. Also on line 62.

Reply:
The number on line 62 needs to be a string, if I read the signature for the function correctly. I removed the quotes in the example and in the test.

Reply:
You're right. my mistake!